app/assets/stylesheets/base/_buttons.scss

```diff
@@ -33,3 +33,9 @@
     }
   }
 }
+
+#get_location{
```

Comment:
I thought font awesome would be useful to differentiate it, that was why I suggested that originally actually. Anyway we can do that later.

I don't really like grey personally but I like how it goes 100%. So I'll change that.